daml/splice-dso-governance/daml/Splice/DsoBootstrap.daml

```diff
@@ -55,10 +56,13 @@ template DsoBootstrap with
         exercise amuletRulesCid AmuletRules_Bootstrap_Rounds with
           round0Duration
           amuletPrice
+          initialRound
         -- create the DSO rules with the SV1 as the sole sv
         let sv1SvInfo = SvInfo with
               name = sv1Name
-              joinedAsOfRound = Round 0
+              joinedAsOfRound = case initialRound of
```

Comment:
wdyt of returning the round from AmuletRules_Bootstrap_Rounds and using that here? That way there is no risk that the actual rounds get out of sync with the joinedAsOfRound field here.